CIScripts/Test/Tests/Integration/DockerDriver.Tests.ps1

```diff
@@ -54,6 +51,15 @@ function Save-DockerDriverUnitTestReport {
 $modules = @("agent")
 
 Describe "Docker Driver" {
```

Comment:
In tests for `Initialize-DockerImage` you have added ` -Tags CI_Systest`. Please add this tag to all system tests.

If it has to be this way, then why?

Reply:
CI selfcheck shouldn't include product systests. Initialize-DockerImage is in utils, so I figured it's part of CI selfcheck.

Reply:
(went with @krdln 's suggestion here)